stellargraph/mapper/graphwave_generator.py

```diff
@@ -169,3 +195,40 @@ def _empirical_characteristic_function(samples, ts):
     embedding = K.flatten(tf.concat([mean_cos_t_psi, mean_sin_t_psi], axis=0))
 
     return embedding
+
+
+def _chebyshev(one_hot_encoded_col, laplacian, coeffs, max_eig):
+    """
+    This function calculates one column of the Chebyshev approximation of exp(-scale * laplacian) for
+    all scales using the approach from: https://arxiv.org/abs/1105.1891. See equations (7)-(11) for more info.
+
+    Args:
+        one_hot_encoded_row (SparseTensor): a sparse tensor indicating which column (node) to calculate.
+        laplacian (SparseTensor): the unnormalized graph laplacian
+        coeffs: the Chebyshev coefficients for exp(-scale * x) for each scale in the shape (num_scales, deg)
+    Returns:
+        (num_scales, num_nodes) tensor of the wavelets for each scale for the specified node.
+    """
+
+    def y(vector):
```

Comment:
I think it would be a good idea to explain why the "variable" `y` is a function here; I'd done that on purpose in my suggestion; although it was rather long-winded there.